week2/2-website/css/style.css

```diff
@@ -15,5 +26,257 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+label {
+  margin: 0 40px 0 0;
+  font-size: 26px;
+  line-height: 70px;
+  display: none;
+}
+#toggle {
+  display: none;
+}
+.all {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.section1 {
+  margin: 0;
+  padding: 0;
+  height: 735px;
+  width: 100%;
+  margin-bottom: 10rem;
+}
+header {
+  background-color: yellowgreen;
+}
+/**************************** The navigator bar style **********************************/
+.nav-bar {
+  position: fixed;
+  display: flex;
+  background-color: white;
+  width: 100%;
+  min-height: 71px;
+}
+/****************************** The container of the list syle************************* */
+.menu-container {
```

Comment:
Rather than setting a percentage width on these - It might be better to use a fixed width.
That way it won't bunch up sooner than it should.
Try `width: 675px;`
Then on `.menu` set `justifiy-content: flex-end` and give each `.menu li` `margin: 0 20px`